packages/ui/src/components/Tabs/style.less

```diff
@@ -53,7 +53,20 @@
     }
   }
 
-  &-nav-operations {
-    display: none !important;
+  &-dropdown {
+    font-family: "Open Sans", var(--condo-font-fallback);
+
+    &-menu {
+      border-radius: 8px;
+
+      &-item-disabled {
+        opacity: @condo-global-opacity-disabled;
+      }
+    }
+
+    & .condo-tabs-tab-label {
+      color: @condo-global-color-gray-7;
+      font-weight: 600;
```

Comment:
It would be better to use `@condo-global-font-weight-semibold` constant from `variables.less`